jenkins/opensearch-dashboards/integ-test.jenkinsfile

```diff
@@ -235,6 +235,27 @@ pipeline {
             }
             post {
                 always {
+                    script {
+                        node(AGENT_LABEL) {
+                            docker.withRegistry('https://public.ecr.aws/') {
+                                docker.image(docker_images["$distribution"]).inside(docker_args["$distribution"]) {
+                                    unstash "integtest-opensearch-dashboards-$BUILD_NUMBER"
+                                    downloadBuildManifest(
+                                        url: BUILD_MANIFEST_URL_OPENSEARCH,
+                                        path: BUILD_MANIFEST_OPENSEARCH
+                                    )
+                                    createUploadTestReportManifest(
```

Comment:
I think there is a file on your hard disk that didn't make it into git, can't find a reference to createUploadTestReportManifest

Reply:
You can see the library here: https://github.com/opensearch-project/opensearch-build-libraries/blob/main/vars/createUploadTestReportManifest.groovy

Reply:
Yes. We have our library in the above link.